addons/purchase_requisition/static/src/views/list/purchase_order_line_compare_list_renderer.js

```diff
@@ -40,6 +40,9 @@ export class PurchaseOrderLineCompareListRenderer extends ListRenderer {
         if (column.name === "price_subtotal" && this.bestFields.best_price_ids.includes(record.resId)) {
         customClassNames.push("text-success");
         }
+        if (column.name === "price_total_cc" && this.bestFields.best_price_ids.includes(record.resId)) {
+        customClassNames.push("text-success");
```

Comment:
You can take the opportunity to rewrite this method (bad indent, multiple `if` statements for the same result, ìf` instead of `else if`, complex way to simply concatenate a string)
```javascript
getCellClass(column, record) {
    let classNames = super.getCellClass(...arguments);
    const { resId } = record;
    const isBestPrice = this.bestFields.best_price_ids.includes(resId)
    if ((column.name === "price_subtotal" && isBestPrice) ||
        (column.name === "price_total_cc" && isBestPrice) ||
        (column.name === "date_planned" && this.bestFields.best_date_ids.includes(resId)) ||
        (column.name === "price_unit" && this.bestFields.best_price_unit_ids.includes(resId))) {
        classNames += " text-success";
    }
    return classNames;
}
```